lib/rototiller/task/collections/command_collection.rb

```diff
@@ -0,0 +1,14 @@
+require 'rototiller/task/collections/param_collection'
+require 'rototiller/task/params/command'
+
+module Rototiller
+  module Task
+
+    class CommandCollection < ParamCollection
+      def allowed_class
+        Command
+      end
```

Comment:
yeah.  that would work.  but i don't see how it's better, nor more clear.  and it's longer